src/core/ext/transport/binder/wire_format/wire_reader_impl.cc

```diff
@@ -306,7 +315,13 @@ absl::Status WireReaderImpl::ProcessStreamingTransactionImpl(
       RETURN_IF_ERROR(parcel->ReadByteArray(&msg_data));
     }
     gpr_log(GPR_INFO, "msg_data = %s", msg_data.c_str());
-    transport_stream_receiver_->NotifyRecvMessage(code, std::move(msg_data));
+    message_buffer_[code] += msg_data;
```

Comment:
I wonder if there exists a hint somewhere (ex. initial metadata) telling us expected total message size? So that we don't need to resize the message buffer

Reply:
I'm not sure if there's any since this hint won't make sense in streaming call scenario.